packages/devextreme/testing/tests/DevExpress.ui.widgets.htmlEditor/htmlEditorParts/toolbarModule.tests.js

```diff
@@ -392,6 +392,20 @@ testModule('Toolbar module', simpleModuleConfig, () => {
         assert.equal(placeholder, 'Test', 'widget has a custom placeholder');
     });
 
+    QUnit.skipInShadowDomMode('textbox should be focused on mousedown (T1196805)', function(assert) {
```

Comment:
Why shouldn't we support it in shadow mode?

Reply:
turned out we should)